CRVResponse/src/CrvCoincidenceFinder_module.cc

```diff
@@ -757,7 +757,7 @@ namespace mu2e
                       [](L &a, L &b){return a->_maxSlope < b->_maxSlope;}))->_maxSlope;
     double maxSlopeDifference = (*std::max_element(layerIterators,layerIterators+n,
                                 [](L &a, L &b){return a->_maxSlopeDifference < b->_maxSlopeDifference;}))->_maxSlopeDifference;
-    double slopes[n-1];
+    std::vector<double> slopes(n-1);
```

Comment:
I'm curious to see, if this impacts the speed, since this part is called very often - especially for events with a large number of hits. Perhaps an std::array may be better than a vector.

Reply:
Array requires knowing the dimensions at compile time

Reply:
Without a proper profiler, I don't think we can accurately answer this question.  The art total CPU time is from the OS, but I'm not sure how accurate it is and whether we could sense the difference in the overall exe time.  The module times art prints are elapsed times and therefore inadequate for small CPU changes.  I don't know what the compiler optimizer is capable of.  If there is a reasonable value for the maximum n, then we can make this a local std::array.  Or you could imagine using a vector created outside of loops, though that is a little non-standard.  I'm willing to do whatever the consensus is.  Ralf, you mentioned an array, do you have a max possible n?

Reply:
The maximum value for n is 4 (=4 CRV layers). This means that the maximum array size is 3.

Reply:
Thanks for the discussion.  Dave is right - I thought I had remembered that the the dimension could be specified at run time so long as it was const and had scope and lifetime that was greater than or equal to the array.  I was wrong about that.   

My sense is that it makes more sense to leave this as std::vector - then it's obvious what the dimension means and it's correct even if a future upgrade puts >4  SiPMS on some counters.  

Sorry for the noise.